sentence-generator.js

```diff
@@ -47,16 +93,18 @@ const sentence = () =>{
         break;
     }
     
-    let FINAL_STRING = sentenceArray.join(' '); //convert sentence array into a string
+    const FINAL_STRING = sentenceArray.join(' '); //convert sentence array into a string
```

Comment:
Only use SCREAMING_SNAKE_CASE for constants like `const MY_VAR = "some string"`. These can be pre-computed before your code is run so they also be kept at the top of the file. 

Since the value of this var doesn't change, I would keep it as `const` but I would use camelCase instead